### ISSUE TITLE
Add manifest and docs for cloud.gov/Cloud Foundry

### ISSUE THREAD

**Issue by yozlet** [18F/generator-18F-static#12]
... for running the site on a static buildpack.

I can take this on if needed.

**Comment by brittag** [comment]
Yes! It would be great for this tool to automatically generate a `manifest.yml`, etc., compatible with the instructions at https://cloud.gov/docs/apps/static/
